pandapower/create.py

```diff
@@ -3943,6 +4005,67 @@ def create_poly_cost(net, element, et, cp1_eur_per_mw, cp0_eur=0, cq1_eur_per_mv
     return index
 
 
+def create_poly_costs(net, elements, et, cp1_eur_per_mw, cp0_eur=0, cq1_eur_per_mvar=0,
+                      cq0_eur=0, cp2_eur_per_mw2=0, cq2_eur_per_mvar2=0, index=None, check=True,
+                      **kwargs):
+    """
+    Creates an entry for polynimoal costs for an element. The currently supported elements are:
+     - Generator ("gen")
+     - External Grid ("ext_grid")
+     - Static Generator ("sgen")
+     - Load ("load")
+     - Dcline ("dcline")
+     - Storage ("storage")
+
+    INPUT:
+        **elements** (iterable of integers) - IDs of the elements in the respective element table
+
+        **et** (string or iterable) - Type of element ["gen", "sgen", "ext_grid", "load", "dcline",
+            "storage"] are possible
+
+        **cp1_eur_per_mw** (float or iterable) - Linear costs per MW
+
+        **cp0_eur=0** (float or iterable) - Offset active power costs in euro
+
+        **cq1_eur_per_mvar=0** (float or iterable) - Linear costs per Mvar
+
+        **cq0_eur=0** (float or iterable) - Offset reactive power costs in euro
+
+        **cp2_eur_per_mw2=0** (float or iterable) - Quadratic costs per MW
+
+        **cq2_eur_per_mvar2=0** (float or iterable) - Quadratic costs per Mvar
+
+    OPTIONAL:
+
+        **index** (int, index) - Force a specified ID if it is available. If None, the index one \
+            higher than the highest already existing index is selected.
+
+        **check** (bool, True) - raises UserWarning if costs already exist to this element.
+
+    OUTPUT:
+        **index** (int) - The unique ID of created cost entry
+
+    EXAMPLE:
+        The polynomial cost function is given by the linear and quadratic cost coefficients.
+
+        create_poly_costs(net, [0, 1], "load", cp1_eur_per_mw = 0.1)
```

Comment:
A more exhaustive example would be helpful. E.g. 
`The polynomial cost function is given by the linear and quadratic cost coefficients. If the first two loads have active power cost functions of the kind c(p) = 0.5 + 1 * p + 0.1 * p^2, the costs are created as follows:
`
`create_poly_costs(net, [0, 1], "load", cp0_eur=0.5, cp1_eur_per_mw = 1, cp2_eur_per_mw2=0.1)`

Reply:
If that is actually a correct way to use this function ;-)